File: spec/code_samples/Java/v1@messages@{messageId}/get.java
String messageId = "messageId2";

messagesController.getMessageStatusAsync(messageId, new APICallBack<GetMessageStatusResponse>() {
    public void onSuccess(HttpContext context, GetMessageStatusResponse response) {
        // TODO success callback handler
    }
    public void onFailure(HttpContext context, Throwable error) {
        // TODO failure callback handler
    }
});

File: spec/code_samples/JavaScript/v1@messages@{messageId}/get.js
let messageId = 'messageId2';

const promise = controller.getMessageStatus(messageId);
promise.then((response) => {
    // this block will be executed on successful endpoint call
    // `response` will be of type 'GetMessageStatusResponse'
}, (err) => {
    // this block will be executed on endpoint call failure
    // `err` is an 'object' containing more information about the error
});